tests/ha/priority_fencing_delay.pm

```diff
@@ -60,6 +60,11 @@ sub run {
         assert_script_run "crm configure clone promotable-1 stateful-1 meta promotable=true";
         assert_script_run "crm resource param stonith-sbd set pcmk_delay_max 15";
         assert_script_run "crm configure property priority-fencing-delay=30";
+        # Workaround for bsc#1244437
+        if (is_sle('>=16')) {
+            assert_script_run "crm cluster restart --all";
```

Comment:
After seeing https://bugzilla.suse.com/show_bug.cgi?id=1244437#c6, we do need a `record_soft_failure` here.

Reply:
I updated the code.